native/lambda_game_engine/src/game.rs

```diff
@@ -31,6 +31,7 @@ pub struct GameConfigFile {
     loot_interval_ms: u64,
     zone_starting_radius: u64,
     zone_modifications: Vec<ZoneModificationConfigFile>,
+    initial_positions: Vec<Position>,
```

Comment:
If you are adding a config field you need to add it to the documantion, probaby this one https://github.com/lambdaclass/lambda_backend_game_engine/blob/main/docs/configuration/game.md